test/data/brave_ads/component/rich_media/manifest.fingerprint

```diff
@@ -0,0 +1 @@
+7e52a6f0da1ce99db948079308c72f97ae96d12c9c2f3fbb10fffdae79f4cd66
```

Comment:
@aseren lets remove `manifest.fingerprint` (this file) because it is not needed for tests.

Reply:
Fixed